package.json

```diff
@@ -16,7 +16,7 @@
         "register-with-browser",
         "workspace-native-window-integration"
     ],
-    "howToCustomFolder":"",
+    "howToCustomFolder":"workspace/vnext",
```

Comment:
Is this correct ?

Reply:
It will go back to blank when it is merged into main so let me update that. It was left like this so test team could still test next branch overnight.